### PR TITLE
Update the swift evolution extension

### PR DESCRIPTION
**Pull request by BProg**
## Description

Update the swift evolution extension to use the new endpoint
Major update to the UI
Add screenshots
Add changelog
Add categories

## Screencast

swift-evolution-1
swift-evolution-5

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Comment by pernielsentikaer** [comment]
Looks like you forgot to run Prettier, do you need help with that @BProg

**Comment by BProg** [comment]
> Looks like you forgot to run Prettier, do you need help with that @BProg

Ran it again on all files, is there anything I missed? I'm running it manually on each file, do you have a script to run it for all files?

File: extensions/swift-evolution/src/ProposalsList.tsx [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+      title={proposal.title}
+      keywords={proposal.keywords}
+      actions={
+        <ActionPanel>
+          <Action
+            title="Read Proposal"
+            icon={Icon.Airplane}
+            onAction={() => push(<ProposalGithubPage markdownUrl={proposal.markdownLink} prUrl={proposal.link} />)}
+          />
+          <Action title="Toggle details" icon={Icon.AlignLeft} onAction={props.toggleDetails} />
```
Comment (pernielsentikaer) [inline extensions/swift-evolution/src/ProposalsList.tsx:122]:
```suggestion
          <Action title="Toggle Details" icon={Icon.AlignLeft} onAction={props.toggleDetails} />
```

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi 👋 

Thanks for your contribution 💪 

I have now tested your extension, and I have some feedback ready for you:

* Could we find a way to group some of the items not to have that many separators?

image



I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

**Comment by BProg** [comment]
> 

Screenshot 2023-01-12 at 10 40 26
I have a small issue, the "Review Manager" title is truncated, and don't know why, I see that in your screenshot, it's fine

**Comment by pernielsentikaer** [comment]
Can you commit your latest code so I can check?

**Comment by BProg** [comment]
Pushed a commit where I removed some separators

**Comment by BProg** [comment]
@pernielsentikaer any luck with the issue, is everything else alright?

**Comment by pernielsentikaer** [comment]
I was able to reproduce the error, and it's something that we'll have to fix on our end - I'll approve and we'll look into a fix soon

**Pull request merged**